.github/pull_request_template.md

```diff
@@ -1,3 +1,7 @@
+## STOP: read before you create the pull request
```

Comment:
This looks a bit too invasive for regular PRs to me, especially that when a PR is ready to be created it means that the change is already in the (public) fork of the author, so it is too late anyway. This even adds extra latency for us to learn about the issue. While the author is figuring out our disclosure procedure, the change is sitting in their fork and we don't know about it (but attackers may).
I am not convinced that this is an improvement to what we already have now.

Reply:
The goal is to improve visibility of our security policy.
There could be security fixes in public forks and we won't have a control over them.
Attackers might go through all public repos or they might miss a few, we don't have a control over them either.
But, what we can do is to make sure that if a contributor wants to push a security PR they are aware of the right process to do so.
From recent events it looks like it is easy to miss the security policy review link while creating a PR so, this change is to just to make sure that contributors go through the security policy before they post a PR.

Reply:
actually a link to the security policy is already there while creating the PR and it is expected for the contributors to go through it. With this change we are just adding a duplicate link to make it more visible and this can help with security fixes from the private repos or for changes that are not added in public forks.

Our security policy page today just asks for the contributor to report the issue to our mailing list and if they like they can encrypt their message. But the policy page itself is so small that it doesn't look like it can add any latency in reporting the issue. If for some reason we think that reviewing the security policy page adds an extra latency for us to learn about the issue then we should revisit the page to improve that.

I can remove the STOP section if it looks invasive but the item in the checklist will at least help us know that the contributor has gone through the security policy and doesn't think it is a security issue.